src/hooks/useDebounce.ts

```diff
@@ -0,0 +1,18 @@
+// eslint-disable-next-line @typescript-eslint/ban-types
+export function useDebounce(callback: Function, delay: number) {
```

Comment:
Custom hooks implies the use of default hooks(like useLocalStorage), which is why they are called custom hooks. And this is a simple function and it should not have a `use `prefix. Please rename it and move to utils for example or use some hooks